contracts/AirdropCampaignData.sol

```diff
@@ -28,7 +30,7 @@ contract AirdropCampaignData is Initializable {
     error AirdropHasFinished();
     error AlreadyFinalized();
 
-    event AdminChanged(address adminAddress);
+    event ContractAdminChanged(address adminAddress);
```

Comment:
@GotchaGuy  This change had to be made due to a warning that the event AdminChanged was declared twice. 
Turns out, there is an event called **AdminChanged** on [a contract we are inheriting from](https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/master/contracts/proxy/ERC1967/ERC1967UpgradeUpgradeable.sol#L117).